src/data/finset/basic.lean

```diff
@@ -795,6 +795,9 @@ by simp only [insert_eq, union_left_comm]
 lemma insert_union_distrib (a : α) (s t : finset α) : insert a (s ∪ t) = insert a s ∪ insert a t :=
 by simp only [insert_union, union_insert, insert_idem]
 
+lemma doubleton_comm (x y : α) : ({x, y} : finset α) = {y, x} :=
```

Comment:
This already exists as `finset.pair_comm`. Please see the comment just below [`has_ssubset`](https://leanprover-community.github.io/mathlib_docs/init/core.html#has_ssubset).

Reply:
Then we should change the name of `finset.card_doubleton` and `list.doubleton_eq`.

Reply:
I added this to our naming guide, https://github.com/leanprover-community/leanprover-community.github.io/pull/276